.ci/deps.opam.sh

```diff
@@ -8,6 +8,7 @@ if [ ! -e ~/infer-linux64-v0.7.0/infer/bin ]; then
   wget -nc -O ~/infer.tar.xz https://github.com/facebook/infer/releases/download/v0.7.0/infer-linux64-v0.7.0.tar.xz
   tar xf ~/infer.tar.xz -C ~/
   cd ~/infer-linux64-v0.7.0
+  apt-get install ocaml ocaml-native-compilers camlp4-extra opam -y
```

Comment:
can this go back to its previous location?
invoking `apt-get` is not cheap; it should only be invoked once.